### ISSUE TITLE
Train on custom dataset

### ISSUE THREAD

**Issue by robosina** [narumiruna/efficientnet-pytorch#30]
Today I tried to use this implementation on custom dataset, as you said in `readme file` for training on custom dataset we have to add path to training dataset by `-r` flag , but in the train.py there is no such flag? So could you please help me about how to train `efficientnet-b0` on custom dataset?

**Comment by narumiruna** [comment]
It seems that I forgot to update `README.md` file. Please change the path to the training dataset in the config file.

**Comment by robosina** [comment]
Thanks , did you mean to add new CustomDataLoader or we can just set path of training data?

**Comment by narumiruna** [comment]
If the directory structure of your dataset is the same as the ImageNet, then you can just set path of training data. Otherwise, you should add a new custom data loader.

**Comment by robosina** [comment]
@narumiruna ,Thanks I add my dataset to the data folder and training is started, but did you get good performance on custom dataset? I have a dataset of two class with images with size 44, but my accuracy don't increase from 78%.

**Comment by narumiruna** [comment]
Maybe you can disable the augmentation by replacing 
https://github.com/narumiruna/efficientnet-pytorch/blob/42e95f57e3423276fff11ce31d5b07b1ce871473/efficientnet/datasets/imagenet.py#L37-L50

with 
```
transform = transforms.Compose([
            transforms.Resize(image_size + 32, interpolation=Image.BICUBIC),
            transforms.CenterCrop(image_size),
            transforms.ToTensor(),
            normalize,
        ])
```
or use less augmentation.

**Comment by robosina** [comment]
Thanks, I will test it
